### PR TITLE
Add setting to show Home on launch

### PR DESCRIPTION
**Pull request by seadowg**
Hey 👋. I absolutely love the app, but I'd prefer it if it showed the Home screen on launch instead of the library screen. I didn't think changing that for everyone was a good idea however - I imagine a lot of folks are used to the existing flow.

This PR adds a setting (in Appearance) that when toggled on, shows the home screen instead of the library screen when the app is launched.

### CODE REVIEW

File: androidApp/main/app/src/main/java/com/simplecityapps/shuttle/ui/screens/main/MainFragment.kt [hunk 1]
```diff
@@ -60,6 +64,9 @@ class MainFragment :
         multiSheetView = view.findViewById(R.id.multiSheetView)
 
         val navController = Navigation.findNavController(requireActivity(), R.id.navHostFragment)
+        if (navController.currentDestination == null) {
+            initializeNavGraph(navController)
```
Comment (seadowg) [inline androidApp/main/app/src/main/java/com/simplecityapps/shuttle/ui/screens/main/MainFragment.kt:68]: Our nav state is persisted after the view is destroyed (when navigating to Settings for example) so we don't to inflate a new nav graph and change the screen.

File: androidApp/main/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -26,4 +26,6 @@
     <string name="dialog_button_enable">Enable</string>
     <!-- Shown once the user enables crash reporting via the 'please consider enabling crash reporting' dialog-->
     <string name="toast_crash_reporting_enabled">Crash reporting enabled</string>
+    <string name="pref_navigation_title">Navigation</string>
+    <string name="pref_show_home_on_launch_title">Show Home on launch</string>
```
Comment (seadowg) [inline androidApp/main/app/src/main/res/values/strings.xml:30]: I'm not sure if this copy is the best. Definitely open to suggestions.

**Comment by timusus** [comment]
Oh S2's first PR!! @seadowg sorry I only just saw this!

**Review by timusus — APPROVED** [review]
Looka great, thanks!

**Pull request merged**